docs/WalletGuide.md

```diff
@@ -49,7 +54,52 @@ There is one more available configuration for wallet that allows to configure in
 For example, to test with local servers on http protocol, http can be manually enabled.
 
 ```kotlin
-val walletCustom = Wallet(StellarConfiguration.Testnet, ApplicationConfiguration(useHttp = true))
+val walletCustom = Wallet(
+  StellarConfiguration.Testnet,
+  ApplicationConfiguration { defaultRequest { url { protocol = URLProtocol.HTTP } } }
+)
+```
+
+### Configuring client
+Wallet SDK uses [ktor client](https://ktor.io/docs/getting-started-ktor-client.html) for all network requests 
+(excluding Horizon, where SDK http client is used). Currently, okhttp engine is configured to be used with the client.  
+Ktor has a reach client configuration, you can read about it more [here](https://ktor.io/docs/create-client.html#configure-client).
+For example, client can be globally configured:
```

Comment:
I think we're missing 'the' in a few places. I also added a few phrasing changes

```suggestion
The Wallet SDK uses the [ktor client](https://ktor.io/docs/getting-started-ktor-client.html) for all network requests 
(excluding Horizon, where the Stellar SDK's http client is used). Currently, the okhttp engine is configured to be used with the client.  
You can read more about how to configure the ktor client [here](https://ktor.io/docs/create-client.html#configure-client).
For example, the client can be globally configured:
```